packages/terra-menu/src/_MenuContent.jsx

```diff
@@ -181,6 +183,22 @@ class MenuContent extends React.Component {
     }
   }
 
+  getSubmenuHeight() {
+    if (this.props.index > 0 && this.listNode) {
+      const bufHeight = (this.context.name || this.context.className) === 'orion-fusion-theme' ? 20 : 10;
```

Comment:
instead of hardcoding the bufHeight can we use `childHeight` itself as bufHeight.

Reply:
https://github.com/cerner/terra-framework/pull/1903#discussion_r1415276136